src/commands/makersuite.rs

```diff
@@ -49,26 +49,26 @@ impl CommandTrait for GoogleGemini {
         }
 
         if let Some(message_image) =
-            telegram_utils::get_message_or_reply_image(&ctx.message, ctx.client_id).await
+            telegram_utils::get_message_or_reply_attachment(&ctx.message, ctx.client_id).await
         {
-            if message_image.file.expected_size > 4 * MEBIBYTE {
-                return Err(CommandError::Custom("the image cannot be larger than 4 MiB.".into()));
+            if message_image.filesize() > 20 * MEBIBYTE {
```

Comment:
is the 20 MiB limit documented, or is this an arbitrary number?

Reply:
arbitrary, but the max is 50 mib since thats how much telegram bots can download

Reply:
the 50 MiB limit is imposed by The Bot API. TDLib has no additional download or upload limits. i think 64 MiB (like 50, except it's a power of 2) is a fair balance between download speed (delay), disk usage, and user experience